src/printer.ts

```diff
@@ -213,9 +213,19 @@ function printNodeNoParens(path: FastPath, options: Options, print: PrintFn) {
                 if (node.init.length) {
                     operator = ' =';
 
-                    right.push(
-                        join(concat([',', line]), path.map(print, 'init'))
-                    );
+                    if (node.init.length > 1) {
```

Comment:
Not really happy with this. Ideally we should always indent but that would break cases like:

```lua
-- before
self.test = function() end

-- after
self.test = function()
        end
```

Reply:
Hmm. I'm not sure what the most idiomatic way of dealing with this would be. In the first example, I think the `-- after` block more clearly breaks up the noise and shows that the result is going into `a`, but that is entirely subjective. Obviously the second example is the more common way and I'd say the `-- before` version looks better.

Reply:
I'm sorry, my example was not clear. What I meant is that counting the `node.init.length` is a hack that does not cover all edge cases:

```lua
----------------------------
-- function self:test() end

-- [OK] indent if node.init.length > 1
function self:test()
end

-- [OK] always indent
function self:test()
end

----------------------------
-- self.test = function() end

-- [OK] indent if node.init.length > 1
self.test = function()
end

-- [WRONG] always indent
self.test = function()
        end

----------------------------
-- local a, b, c, d, e, f = emptyFunction(function() return true end)

-- [WRONG] indent if node.init.length > 1
local a,
    b,
    c,
    d,
    e,
    f =
    emptyFunction(
    function()
        return true
    end
)

-- [OK] always indent:
local a,
    b,
    c,
    d,
    e,
    f =
    emptyFunction(
        function()
            return true
        end
    )
```

Which is why I'm not quite happy with it.